build.py

```diff
@@ -1,18 +1,69 @@
-import math
-import subprocess
 from pathlib import Path
 
+import argparse
+
 import fontmake.instantiator
 import fontTools.designspaceLib
 import ufo2ft
 import ufoLib2
 import vttLib
-import copy
 
 INPUT_DIR = Path("sources")
 OUTPUT_DIR = Path("build")
+VTT_DATA_FILE = (INPUT_DIR / "vtt_data" / "CascadiaCode.ttx")
+
+def step_set_font_name(n):
+    def _set(instance):
+        instance.info.familyName = n
+    return _set
+
+def step_merge_glyphs_from_ufo(path):
+    def _merge(instance):
+        ufo = ufoLib2.objects.font.Font.open(path)
```

Comment:
```suggestion
        ufo = ufoLib2.Font.open(path)
```

Reply:
LOL. Thanks.

I'm not a Python person, so do expect to find some weird things like this that I've missed. :smile:

Reply:
Dealing with font building pipelines will make you intimately familiar with Python, whether you want it or not :feelsgood: